roles/openshift_storage_glusterfs/templates/gluster-block-storageclass.yml.j2

```diff
@@ -9,7 +9,7 @@ metadata:
 {% endif %}
 provisioner: gluster.org/glusterblock
 parameters:
-  resturl: "http://{% if glusterfs_heketi_is_native %}{{ glusterfs_heketi_route }}{% else %}{{ glusterfs_heketi_url }}:{{ glusterfs_heketi_port }}{% endif %}"
+  resturl: "http://{% if glusterfs_heketi_is_native %}heketi-storage.glusterfs.svc.cluster.local:8080{% else %}{{ glusterfs_heketi_url }}:{{ glusterfs_heketi_port }}{% endif %}"
```

Comment:
You can't just assume this will be the actual name of the service. It should be something like:
```
heketi-{{ glusterfs_name }}.{{ glusterfs_namespace }}.svc
```
I'll note that I'm not sure how constant `cluster.local` is, I believe it can be configured, so we may be better off omitting it.